reference/config_files/profiles.rst

```diff
@@ -22,16 +22,27 @@ They have this structure:
     [options]
     MyLib:shared=True
 
-    [buildenv]
-    VAR1=value
-
     [tool_requires]
     tool1/0.1@user/channel
     *: tool4/0.1@user/channel
 
+    [system_tools]
```

Comment:
This has been renamed to ``platform_tool_requires``, this is why I suggest to do it both in the same PR